reference/debugging.md

```diff
@@ -0,0 +1,300 @@
+# Debugging
+
+If you’re experiencing issues with Ambassador and cannot diagnose the issue through the "Diagnostics" tab from the [Edge Policy Console](../../about/edge-policy-console), this document covers various approaches and advanced use cases for debugging Ambassador issues.
+
+First, create an example configuration for debugging demonstrations that are available throughout this document.
+
+Then, complete the following debugging options:
+
+* Check Ambassador Status
+* Review Ambassador Logs
+* Examine Pod and Container Contents
+* Customize the Envoy Configuration
+
+## Example Configuration
+
+To see a demonstration of how you can debug your Ambassador instance, follow the instructions to create an example mapping configuration.
+
+Note: The following assumes that you deployed Ambassador and the following services from the [quick start installation guide](../../user-guide/install) to a Kubernetes cluster.
+
+**In the command line:**
+
+1. Create a cluster in GKE with RBAC support enabled, and your user account configured correctly. Then run the following:
+
+    ```shell
```

Comment:
I think this should be `console`.  The difference is that `shell` highlights input-to-a-shell, where `console` highlights a recording of a shell session, including output and prompts and such. (Put another way: use `shell` if it says `kubectl …`; use `console` if it says `$ kubectl …`)

```suggestion
    ```console
```